testing/run-scripts/run_cloud.sh

```diff
@@ -57,14 +60,18 @@ fi
 #  - server's hostname
 if [ -z "$BANNER" ]
 then
-  if docker ps -a | grep uproxy-sshd >/dev/null
-  then
-    if [ `docker inspect --format='{{ .State.Status }}' uproxy-sshd` != "running" ]
+    if docker ps -a | grep uproxy-sshd >/dev/null
     then
-      docker start uproxy-sshd > /dev/null
+        if [ `docker inspect --format='{{ .State.Status }}' uproxy-sshd` != "running" ]
```

Comment:
Yup! Two spaces would be my vote, keep it the same as our TS code. No need to change this pull request now...I can do a one-time followup.